collections/catalog/Add Interface by Template.bru

```diff
@@ -0,0 +1,33 @@
+
+meta {
+  name: Add e-service interface document by template
```

Comment:
```suggestion
  name: Add e-service template instance interface document
```

Reply:
handled with [c212232](https://github.com/pagopa/interop-be-monorepo/pull/1620/commits/c212232e6615a52174a8b71226d1609061ae7316)